tools/release/lp_build_monitor_recipe.py

```diff
@@ -33,7 +33,15 @@ def start_all_build(build_recipe):
             print("An identical build of this recipe is already pending")
 
 
-def wait_every_build_started(build_recipe):
+def are_binary_builds_ongoing(build_recipe):
+    # Note: this is a different definition of pending, this means pending or
```

Comment:
different compared to?
I'd use a non-referential clarification here, i.e.

>The pending attribute used here means ...

Reply:
Sadly, compared to anywhere else in launchpadlib, included the `buildstatus` in the docstring below. I can change this to your version, it is clearer, I was a bit salty I think